package.json

```diff
@@ -11,7 +11,7 @@
     "firefoxos",
     "testing"
   ],
-  "version": "2.0.0-beta.6",
+  "version": "2.0.0-beta.7",
```

Comment:
should the version be bumped as a part of this PR?

Reply:
i already published this version having forgotten to push the rev change, so it does not need to be a part of this PR but it is a moot point, so i am just getting it in now.